### DISCUSSION TITLE
Welcome to godot-engine.jwt Discussions!

### DISCUSSION THREAD

**Discussion by fenix-hub** [fenix-hub/godot-engine.jwt#42] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by srsizan** [comment]
res://addons/godot-engine.jwt-main/addons/jwt/src/JWT.gd:1 - Parse Error: Could not find base class "Reference".
  modules/gdscript/gdscript.cpp:2705 - Failed to load script "res://addons/godot-engine.jwt-main/addons/jwt/src/JWTAlgorithm.gd" with error "Parse error". (User)
  modules/gdscript/gdscript.cpp:2705 - Failed to load script "res://addons/godot-engine.jwt-main/addons/jwt/src/JWTAlgorithmBuilder.gd" with error "Parse error". (User)
